mantis-common/src/main/java/io/mantisrx/common/Ack.java

```diff
@@ -24,4 +24,5 @@
  */
 @Value(staticConstructor = "getInstance")
 public class Ack implements Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
nit: can you add a link to the doc about how this interact with JVM?

Reply:
+1